apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js

```diff
@@ -1,24 +1,26 @@
-import React from 'react'
-import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader'
-import axios from "axios";
+import React from 'react';
+import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader';
+import axios from 'axios';
 import { atoms } from '@devlaunchers/components/src/components';
 import IdeaCard from '../../common/IdeaCard/IdeaCard';
 import BackButton from '../../common/BackButton/BackButton';
 import Dropdown from '@devlaunchers/components/components/organisms/Dropdown';
+import useResponsive from '@devlaunchers/components/src/hooks/useResponsive';
```

Comment:
I see that you guys are using the hooks we provided without us even telling about them, I hope it works well and if not please always create issues with the label platform enablement :D

Reply:
Thanks Mohammed haha that was a recommendation from Dave, I'm not sure how he found out that hook existed

Reply:
I learned it from Anwar (: